core/server/data/schema/default-settings.json

```diff
@@ -106,5 +106,10 @@
         "password": {
             "defaultValue": ""
         }
+    },
+    "notifications": {
+        "seenNotifications": {
+            "defaultValue": "[]"
+        }
```

Comment:
Little thing - I think this belongs up in core with the other notification-related settings? Unless there's a strong need for a new `type` here that I've missed?